numpy/core/tests/test_multiarray.py

```diff
@@ -2248,6 +2248,15 @@ def test__deepcopy__(self, dtype):
         with pytest.raises(AssertionError):
             assert_array_equal(a, b)
 
+    def test__deepcopy__catches_failure(self):
+        class MyObj:
+             def __deepcopy__(self, *args, **kwargs):
+                 return Exception
```

Comment:
ooops, `raise Exception`, did I write that? :)

Reply:
Oh yikes, that's a typo on my end; will fix :sweat_smile: though I still do have a question below I'm confused about